src/models/evento.js

```diff
@@ -30,25 +29,18 @@ const EventoSchema = Schema({
         required: true
     },
     fecha: {
-        type: String,
+        type: Date,
         required: true
     },
     enlace: {
         type: String,
         required: true
     },
-    proveedor: {
-        required: true,
+    profesional: {
         type: Schema.Types.ObjectId,
-        ref: 'Proveedor'
+        ref: 'Usuario'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Backend/pullRequest?prid=7157741)
```suggestion
        ref: "Usuario"
```